app/src/main/java/mdb/project2/pokedex/PokemonProfile.java

```diff
@@ -0,0 +1,86 @@
+package mdb.project2.pokedex;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.graphics.PorterDuff;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import org.w3c.dom.Text;
+
+import java.text.Normalizer;
+import java.util.ArrayList;
+
+public class PokemonProfile extends AppCompatActivity {
+    private Pokedex pokedex = new Pokedex();
+    private ArrayList<Pokedex.Pokemon> pokeList = pokedex.getPokemon();
+    private Pokedex.Pokemon pokemon = null;
+    private ImageView pokeProfile;
+    private TextView profileName,
+            profileNumber,
+            profileAttack,
+            profileDefense,
+            profileHp,
+            profileSpecies;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_pokemon_profile);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        Intent intent = getIntent();
+        String name = intent.getStringExtra("NAME");
```

Comment:
For keys, always store the key in a static final string so changes to keys are easy to make instead of CTRL+F and digging through piles of crap